fig/service.py

```diff
@@ -329,7 +329,14 @@ def start_container(self, container, intermediate_container=None, **override_opt
             dns_search=dns_search,
             restart_policy=restart,
             cap_add=cap_add,
-            cap_drop=cap_drop,
+            cap_drop=cap_drop
+        )
+
+    def start_container(self, container=None, intermediate_container=None, **override_options):
+        container = container or self.create_container(**override_options)
```

Comment:
This reverts an earlier change. `container` should be a required arg, not a kwarg, and this line should be removed.